plugins/bulk.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: bulk
```

Comment:
maybe consider a name that's a bit more specific (e.g. `bulk-action`, `bulk-resources`, `bulk-resource-action`), [per the naming guide](https://github.com/GoogleContainerTools/krew/blob/master/docs/NAMING_GUIDE.md#be-specific).

Reply:
I renamed plugin as bulk-action.